Document missing wallet RPC fields (transfer/sweep results, multisig flags, …) (#340)
monero-project/monero-docs: docs/en/rpc-library/wallet-rpc.md

Alias:  _None_.

Inputs:

> -   _autosave_current_  - boolean; When true, stores (saves) the currently open wallet via store() before it is closed and deleted.

Reviewer: ```suggestion
-   _autosave_current_  - boolean; (Optional; Default = true) When true, stores (saves) the currently open wallet via store() before it is closed.
```

ive never seen this delete a wallet. Wheres that come from?

Author: You're right, it doesn't - that was my misread of the close handler (it unloads the wallet from memory, it doesn't delete the file). Dropped "and deleted" and applied your suggestion.

Inputs:

-   _info_  - array of string; List of multisig info in hex format from other participants.

> -   _refresh_after_import_  - boolean; When set to true (the default), the wallet refreshes after importing the multisig info, rescanning spent outputs if the daemon is trusted.

Reviewer: ```suggestion
-   _refresh_after_import_  - boolean; (Optional; Default = true) When set to true, the wallet refreshes after importing the multisig info, rescanning spent outputs if the daemon is trusted.
```

Author: Applied, with one tweak: I dropped the "rescanning spent outputs if the daemon is trusted" clause - that behavior doesn't actually happen on this path, so it was misleading. Left it as just "the wallet refreshes after importing the multisig info."

-   _filename_  - string; wallet name stored in --wallet-dir.
-   _password_  - string; (Optional) only needed if the wallet has a password defined.

> -   _autosave_current_  - boolean; When set to true (the default), if a wallet is already open it is stored (saved) before the requested wallet file is opened.

Reviewer: ```suggestion
-   _autosave_current_  - boolean; (Optional; Default = true) When true, if a wallet is already open it will be stored (saved) before the requested wallet file is opened.
```

Author: Applied, thanks.

Alias:  _None_.

Inputs:

> -   _enable_multisig_experimental_  - boolean; When true, enables experimental multisig support on the wallet before preparing the multisig info.

Reviewer: Optional? Default?

Author: Optional, default false. Annotation added.

Alias:  _None_.

Inputs:

> -   _hard_  - boolean; When true, performs a hard rescan of the blockchain; passed directly to the wallet's rescan_blockchain.

Reviewer: ```suggestion
-   _hard_  - boolean; (Optional; Default = false) If true, you will lose any information which can not be recovered from the blockchain.
```

Author: Applied, thanks.
